packages/react-components/react-tags/stories/TagGroup/TagGroupDismiss.stories.tsx

```diff
@@ -3,10 +3,10 @@ import { TagGroup, Tag, TagButton, TagButtonProps, TagProps, TagGroupProps } fro
 
 export const Dismiss = () => {
   const defaultItems = [
-    { value: '1', children: 'Tag 1' },
-    { value: '2', children: 'Tag 2' },
-    { value: 'tagButton-foo', children: 'Foo' },
-    { value: 'tagButton-bar', children: 'Bar' },
+    { value: '1', children: 'Tag 1', 'aria-label': 'Tag1, remove' },
+    { value: '2', children: 'Tag 2', 'aria-label': 'Tag2, remove' },
```

Comment:
I know we didn't get into this in the call and I don't think this should hold up this PR, but I think a better approach would probably be to have a default `aria-label` of "Remove" for the "x" icon. That shouldn't hold up this PR, just something to look at before making Tag/TagButton stable

Reply:
I'm not sure about setting the default 'remove' because of internationalization. Looking at other components, we were doing this: https://github.com/microsoft/fluentui/blob/0f5a9008e78d8a3e89ebdecb26c48694b30d35f4/packages/react-components/react-avatar/src/components/AvatarGroupPopover/useAvatarGroupPopover.tsx#L88 But I wonder if i18n were considered.